docs/guides/docs/running-a-node/running-a-local-node.mdx

```diff
@@ -47,7 +45,7 @@ Here is an example of what that looks like using version {props.fuelCoreVersion}
 To start the node with a custom configuration, you can use the command below:
```

Comment:
This is outdated: "To modify the initial state of the chain, you must configure a `chainConfig.json` file."